.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "interactive"
```

Comment:
can we put .vscode in a .gitignore file

Reply:
@maeveho25 this means that you will need to remove this file from the PR